digitalstrategie/urls.py

```diff
@@ -17,23 +18,27 @@
     path('admin/', include(wagtailadmin_urls)),
     path('documents/', include(wagtaildocs_urls)),
     path('i18n/', include('django.conf.urls.i18n')),
-    re_path(r'^sitemap\.xml$', wagtail_sitemap),
     re_path(r'^robots\.txt$', TemplateView.as_view(
         template_name='robots.txt',
         content_type="text/plain"), name="robots_file"),
-    path('search/', SearchResultsView.as_view(), name="search"),
     path('newsletter-signup/', newsletter_view, name="newsletter-signup"),
 
     # For anything not caught by a more specific rule above, hand over to
     # Wagtail's page serving mechanism. This should be the last pattern in
     # the list:
-    path('', include(wagtail_urls)),
+    # path('', include(wagtail_urls)),
 
     # Alternatively, if you want Wagtail pages to be served from a subpath
     # of your site, rather than the site root:
     #    url(r'^pages/', include(wagtail_urls)),
```

Comment:
Can we either move or entirely remove this?

Reply:
I guess? I wasnt sure where this is coming from, so just commented out.. so should I remove it?